go-controller/pkg/cni/helper_linux.go

```diff
@@ -169,17 +169,7 @@ func setupSriovInterface(netns ns.NetNS, containerID, ifName string, ifInfo *Pod
 	contIface := &current.Interface{}
 
 	// 1. get VF netdevice from PCI
```

Comment:
The comment is no longer accurate.

Reply:
yes, i have updated the comment. I didln't want to remove it, then I will need to re-number all of the other comments.